💖 Add WebRTC Card

ankostis opened the pull request:
# Describe the proposed change

The *WebRTC Camera* project offers a new card+intergation for viewing IP cameras RTSP streams in real time using WebRTC and MSE technology suppoorting Pan/Titlt controls.


## The link

[Add the link here.](https://github.com/AlexxIT/WebRTC)

## The annoying "I agree" thing

**PRO TIP!** _Don't check the boxes right now! Open the PR first, and it will allow you actually to check the boxes using simple mouse clicks._

- [x] Check this box if you have read, understand, comply, and agree with our [Code of Conduct](https://github.com/frenck/awesome-home-assistant/blob/master/CODE_OF_CONDUCT.md).

- [x] Check this box if you have read, understand, and comply with our [Contribution Guidelines](https://github.com/frenck/awesome-home-assistant/blob/master/CONTRIBUTING.md).
---
Anyone who agrees with this pull request could vote for it by adding a :+1: to it, and usually, the maintainer will merge it when votes reach significant numbers.

ankostis committed:
WebRTC Card

For vewing IP cameras RTSP streams in real time using WebRTC and MSE technology suppoorting Pan/Titlt controls.

ristomatti commented:
I think this should go under the "Custom Components" section and the link text should be just "WebRTC" as is the title of the component. I would also suggest adding "real-time" as part of the description.

This integration blew me mind of what is possible and definitely should be on this list.

ankostis commented:
Have not received any review comment - have you "submitted the review"?
Please reopen, so that i can fix this.

frenck commented:
![image](https://user-images.githubusercontent.com/195327/173521790-12b4ebb3-9ca8-4a92-83c8-c99afddb6a1c.png)

ankostis committed:
WebRTC: fix md and recategorize

- Fixed link md-syntax.
- Recategorize from LovelaceUI cards-->Custom components.
- Noted that RTSP means real-time.
- Applied review comment on #332

ankostis commented:
My apologies for having missed this was a review comment.

> I think this should go under the "Custom Components" section 

Ok, commited.
Note that it is actually just a LovelaceUI card, installed as a HACS extension.

frenck commented on README.md:
```diff
@@ -266,6 +265,7 @@ _Additional components for Home Assistant, that were created by the community._
 - [Google Geocode](https://github.com/michaelmcarthur/GoogleGeocode-HASS) - Converts a device tracker location into a human-readable address.
 - [Lutron Caseta Pro](https://github.com/upsert/lutron-caseta-pro) - Integrates Lutron Caseta Smart Bridge PRO / RA2 Select.
 - [SmartIR](https://github.com/smartHomeHub/SmartIR) - Integrates devices using Broadlink IR.
+- [WebRTC Camera](https://github.com/AlexxIT/WebRTC) - view RTSP streams from IP-Cameras on real-time through WebRTC or MSE with Pan/Zoom controls.
```
This is not an ordered list, please move it to the bottom.

frenck commented:
> Note that it is actually just a LovelaceUI card, installed as a HACS extension.

I don't think so? Are you mixing up repositories? This is a custom integration, see also the repository which has an `custom_component` directory with integration Python code in it: <https://github.com/AlexxIT/WebRTC/tree/master/custom_components/webrtc>

ristomatti commented:
> My apologies for having missed this was a review comment.
> 
> >

Obviously it was not, rather a suggestion from someone coming across the repo the first time.

Anyhow, great to see activity on this repo as I thought it had been forgotten. Not that I didn't know frenck has a ton of likely more worthy stuff going on.

ankostis committed:
WebRTC: reorder new item to the bottom

ankostis commented:
> Obviously it was not [a commit review] ...

So i miss the meaning of things, big time.

> ... frenck has a ton of likely more worthy stuff going on.

 ¯\_(ツ)_/

frenck committed:
Language tweaks

frenck committed:
Merge branch 'main' into patch-1

frenck reviewed (approved):
Thanks, @ankostis 👍
